RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -43,50 +51,63 @@ import RevenueCat
         }
     }
     @Published
-    var configuration: CustomerCenterConfigData?
+    var configuration: CustomerCenterConfigData? {
+        didSet {
+            guard let currentVersionString = currentAppVersion?.versionString() else {
+                self.appIsLatestVersion = defaultAppIsLatestVersion
+                return
+            }
+            guard let latestVersionString = configuration?.lastPublishedAppVersion?.versionString() else {
+                self.appIsLatestVersion = defaultAppIsLatestVersion
+                return
+            }
+
+            do {
+                let currentVersion = try SemanticVersion(currentVersionString)
+                let latestVersion = try SemanticVersion(latestVersionString)
+                self.appIsLatestVersion = currentVersion >= latestVersion
+            } catch {
+                self.appIsLatestVersion = defaultAppIsLatestVersion
+            }
```

Comment:
You can combine all of these by chaining the conditions like

```swift
didSet {
    guard
        let currentVersionString = currentAppVersion?.versionString(),
        let latestVersionString = configuration?.lastPublishedAppVersion?.versionString(),
        let currentVersion = try? SemanticVersion(currentVersionString),
        let latestVersion = try? SemanticVersion(latestVersionString)
    else {
        self.appIsLatestVersion = defaultAppIsLatestVersion
        return
    }

    self.appIsLatestVersion = currentVersion >= latestVersion
}
```

Reply:
Very nice, thanks! [19b7881](https://github.com/RevenueCat/purchases-ios/pull/4169/commits/19b78817d6675482ef93a1a8b48a0c6874985a53)